app/src/main/kotlin/com/wire/android/ui/common/MoreOptionIcon.kt

```diff
@@ -1,24 +1,38 @@
 package com.wire.android.ui.common
 
+import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.foundation.layout.size
 import androidx.compose.material3.Icon
-import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
 import com.wire.android.R
-import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.common.button.IconAlignment
+import com.wire.android.ui.common.button.WireTertiaryButton
+import com.wire.android.ui.common.button.wireSecondaryButtonColors
 
 @Composable
 fun MoreOptionIcon(
```

Comment:
Isn't it a identical button to the CopyButton? Can we create a generic reusable one and just change the content inside?